pgx-pg-sys/src/lib.rs

```diff
@@ -160,29 +162,38 @@ pub use internal::pg13::*;
 #[cfg(feature = "pg14")]
 pub use internal::pg14::*;
 
-/// A trait applied to all of Postgres' `pg_sys::Node` types and its subtypes
+/// A trait applied to all Postgres `pg_sys::Node` types and subtypes
 pub trait PgNode {
-    type NodeType;
+    // TODO(0.6.0): seal this trait or take this trait private?
```

Comment:
It seems unlikely we have any dependents on this but I didn't want to scratch it off right away as someone may be using it as a trait bound or something.